firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/HarmCategory.kt

```diff
@@ -33,5 +33,8 @@ public class HarmCategory private constructor(public val ordinal: Int) {
 
     /** Dangerous content. */
     @JvmField public val DANGEROUS_CONTENT: HarmCategory = HarmCategory(4)
+
+    /** Content that may be used to harm civic integrity. */
```

Comment:
We have the best javadocs around 🥇

Reply:
Directly from our iOS friends

https://github.com/firebase/firebase-ios-sdk/blob/7bdf3bb7bdc6836cde24a2c1c6ca53e4a76aa85a/FirebaseVertexAI/Sources/Safety.swift#L253